### PR TITLE
Update tutorials for planning pipeline refactoring

### PR DESCRIPTION
**Pull request by sjahr**
### Description

Updates tutorials to changes from https://github.com/ros-planning/moveit2/pull/2429

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/examples/planning_adapters/planning_adapters_tutorial.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+Planning Request Adapters is a concept in MoveIt which can be used to modify the trajectory (pre-processing and/or post-processing) for a motion planner. Some examples of existing planning adapters in MoveIt include AddTimeParameterization, FixWorkspaceBounds, FixStartBounds, FixStartStateCollision, CHOMPOptimizerAdapter, etc. ! Using the concepts of Planning Adapters, multiple motion planning algorithms can be used in a pipeline to produce robust motion plans. For example, a sample pipeline of motion plans might include an initial plan produced by OMPL which can then be optimized by CHOMP to produce a motion plan which would likely be better than a path produced by OMPL or CHOMP alone. Similarly, using the concept of Planning Adapters, other motion planners can be mixed and matched depending on the environment the robot is operating in. This section provides a step by step tutorial on using a mix and match of different motion planners and also provides insights on when to use which particular motion planners.
```
Comment (MarqRazz) [inline doc/examples/planning_adapters/planning_adapters_tutorial.rst:9]: Is the `!` needed here 
```
etc. !
```

Another good example would be running TOTG then Ruckig to reduce jerk.

File: doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst [hunk 2]
```diff
@@ -98,31 +98,3 @@ OMPL is a open source library for sampling based / randomized motion planning al
...
 
 For scenes containing obstacles, CHOMP often generates paths which do not prefer smooth trajectories by addition of some noise (*ridge_factor*) in the cost function for the dynamic quantities of the robot (like acceleration, velocity). CHOMP is able to avoid obstacles in most cases, but it can fail if it gets stuck in local minima due to a bad initial guess for the trajectory. OMPL can be used to generate collision-free seed trajectories for CHOMP to mitigate this issue.
-
-Using CHOMP as a post-processor for OMPL
```
Comment (MarqRazz) [inline doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst:102]: Why is this section removed? In the example I commented above it was the use case for post-processing.

File: doc/examples/planning_adapters/planning_adapters_tutorial.rst (hunk 1 above)
Comment (sea-bass) [inline doc/examples/planning_adapters/planning_adapters_tutorial.rst:9]: Please change to one sentence per line. This paragraph is so hard to read.

Also, this paragraph starts with "planning request adapters" but we're making these more generic to be both request and response.

File: doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst (hunk 2 above)
Comment (sea-bass) [reply doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst:102]: I think in the next part of these PRs, CHOMP will be changed from an adapter to an actual chain of planning pipelines.

**Comment by tylerjw** [comment]
Landed the dependent change in MoveIt; let's finish this one up and get it landed.

File: doc/examples/planning_adapters/planning_adapters_tutorial.rst (hunk 1 above)
Comment (sjahr) [reply doc/examples/planning_adapters/planning_adapters_tutorial.rst]: This tutorial is still targeting ROS 1. To reduce the scope of this PR, I'll not touch it and removed the changes that did so

File: doc/concepts/motion_planning.rst [hunk 3]
```diff
@@ -47,36 +47,30 @@ Pre-processing is useful in several situations, e.g. when a start state for the
...
-FixWorkspaceBounds
+ValidateWorkspaceBounds
 ^^^^^^^^^^^^^^^^^^
 
 The fix workspace bounds adapter will specify a default workspace for planning: a cube of size 10 m x 10 m x 10 m.
 This workspace will only be specified if the planning request to the planner does not have these fields filled in.
 
-FixStartStateCollision
+CheckStartStateCollision
 ^^^^^^^^^^^^^^^^^^^^^^
```
Comment (sjahr) [inline doc/concepts/motion_planning.rst:68]:
```suggestion
CheckStartStateCollision
^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by sjahr** [comment]
Closing this in favor of https://github.com/ros-planning/moveit2_tutorials/pull/834

**Pull request closed without merging**